src/components/com_tjcertificate/administrator/controllers/certificates.php

```diff
@@ -34,4 +38,100 @@ public function getModel($name = 'Certificate', $prefix = 'TjCertificateModel')
 	{
 		return parent::getModel($name, $prefix, array('ignore_request' => true));
 	}
+
+	/**
+	 * Method to delete the certificate.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function deleteCertificate()
+	{
+		$user = Factory::getUser();
+		$app  = Factory::getApplication();
+		$certificateId = $app->input->getInt('id');
+		$model = $this->getModel();
+
+		if (!$user->authorise('certificate.external.manage', 'com_tjcertificate'))
+		{
+			$checkCertificateExist = $model->checkCertificateExist($certificateId, $user->id);
+
+			if ($checkCertificateExist === false)
+			{
+				return false;
+			}
+		}
```

Comment:
are we not allowing front-end user to delete their own certificates?

Reply:
Yes, we are allowing